[INFO] 19 Jul 09:57:23.822 AM WrapperSimpleAppMain [org.quartz.impl.StdSchedulerFactory]
Using default implementation for ThreadExecutor

[INFO] 19 Jul 09:57:24.054 AM WrapperSimpleAppMain [org.quartz.simpl.SimpleThreadPool]
Job execution threads will use class loader of thread: WrapperSimpleAppMain

[INFO] 19 Jul 09:57:24.094 AM WrapperSimpleAppMain [org.quartz.core.SchedulerSignalerImpl]
Initialized Scheduler Signaller of type: class org.quartz.core.SchedulerSignalerImpl

[INFO] 19 Jul 09:57:24.098 AM WrapperSimpleAppMain [org.quartz.core.QuartzScheduler]
Quartz Scheduler v.2.1.7 created.

[INFO] 19 Jul 09:57:24.100 AM WrapperSimpleAppMain [org.quartz.simpl.RAMJobStore]
RAMJobStore initialized.

[INFO] 19 Jul 09:57:24.101 AM WrapperSimpleAppMain [org.quartz.core.QuartzScheduler]
Scheduler meta-data: Quartz Scheduler (v2.1.7) 'DefaultQuartzScheduler' with instanceId 'NON_CLUSTERED'
  Scheduler class: 'org.quartz.core.QuartzScheduler' - running locally.
  NOT STARTED.
  Currently in standby mode.
  Number of jobs executed: 0
  Using thread pool 'org.quartz.simpl.SimpleThreadPool' - with 10 threads.
  Using job-store 'org.quartz.simpl.RAMJobStore' - which does not support persistence. and is not clustered.


[INFO] 19 Jul 09:57:24.101 AM WrapperSimpleAppMain [org.quartz.impl.StdSchedulerFactory]
Quartz scheduler 'DefaultQuartzScheduler' initialized from default resource file in Quartz package: 'quartz.properties'

[INFO] 19 Jul 09:57:24.101 AM WrapperSimpleAppMain [org.quartz.impl.StdSchedulerFactory]
Quartz scheduler version: 2.1.7

[INFO] 19 Jul 09:57:24.138 AM WrapperSimpleAppMain [org.quartz.core.QuartzScheduler]
Scheduler DefaultQuartzScheduler_$_NON_CLUSTERED started.

[ERROR] 19 Jul 11:54:36.794 AM Thread-2 [org.apache.thrift.ProcessFunction]
Internal error processing addPlanTarget

redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
	at redis.clients.util.Pool.getResource(Pool.java:22)
	at org.cz.epm.data.manager.RedisManager.getJedis(RedisManager.java:31)
	at org.cz.epm.data.manager.RedisManager.HGet(RedisManager.java:36)
	at org.cz.epm.resource.Mapper.getMapKeyValue(Mapper.java:45)
	at org.cz.epm.resource.Mapper.GetMapKey(Mapper.java:19)
	at org.cz.epm.thrift.service.EpmDatahouseImpl.addPlanTarget(EpmDatahouseImpl.java:90)
	at org.cz.epm.thrift.generated.Datahouse$Processor$addPlanTarget.getResult(Datahouse.java:1388)
	at org.cz.epm.thrift.generated.Datahouse$Processor$addPlanTarget.getResult(Datahouse.java:1)
	at org.apache.thrift.ProcessFunction.process(ProcessFunction.java:39)
	at org.apache.thrift.TBaseProcessor.process(TBaseProcessor.java:39)
	at org.apache.thrift.server.AbstractNonblockingServer$FrameBuffer.invoke(AbstractNonblockingServer.java:478)
	at org.apache.thrift.server.TNonblockingServer.requestInvoke(TNonblockingServer.java:115)
	at org.apache.thrift.server.AbstractNonblockingServer$AbstractSelectThread.handleRead(AbstractNonblockingServer.java:209)
	at org.apache.thrift.server.TNonblockingServer$SelectAcceptThread.select(TNonblockingServer.java:198)
	at org.apache.thrift.server.TNonblockingServer$SelectAcceptThread.run(TNonblockingServer.java:154)
Caused by: redis.clients.jedis.exceptions.JedisConnectionException: java.net.ConnectException: Connection refused
	at redis.clients.jedis.Connection.connect(Connection.java:134)
	at redis.clients.jedis.BinaryClient.connect(BinaryClient.java:69)
	at redis.clients.jedis.BinaryJedis.connect(BinaryJedis.java:1665)
	at com.trigonic.jedis.NamespaceJedis.connect(NamespaceJedis.java:104)
	at com.trigonic.jedis.NamespaceJedisFactory.makeObject(NamespaceJedisFactory.java:33)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1188)
	at redis.clients.util.Pool.getResource(Pool.java:20)
	... 14 more
Caused by: java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at redis.clients.jedis.Connection.connect(Connection.java:129)
	... 20 more
[ERROR] 19 Jul 11:54:42.227 AM Thread-2 [org.apache.thrift.ProcessFunction]
Internal error processing addPlanTarget

redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
	at redis.clients.util.Pool.getResource(Pool.java:22)
	at org.cz.epm.data.manager.RedisManager.getJedis(RedisManager.java:31)
	at org.cz.epm.data.manager.RedisManager.HGet(RedisManager.java:36)
	at org.cz.epm.resource.Mapper.getMapKeyValue(Mapper.java:45)
	at org.cz.epm.resource.Mapper.GetMapKey(Mapper.java:19)
	at org.cz.epm.thrift.service.EpmDatahouseImpl.addPlanTarget(EpmDatahouseImpl.java:90)
	at org.cz.epm.thrift.generated.Datahouse$Processor$addPlanTarget.getResult(Datahouse.java:1388)
	at org.cz.epm.thrift.generated.Datahouse$Processor$addPlanTarget.getResult(Datahouse.java:1)
	at org.apache.thrift.ProcessFunction.process(ProcessFunction.java:39)
	at org.apache.thrift.TBaseProcessor.process(TBaseProcessor.java:39)
	at org.apache.thrift.server.AbstractNonblockingServer$FrameBuffer.invoke(AbstractNonblockingServer.java:478)
	at org.apache.thrift.server.TNonblockingServer.requestInvoke(TNonblockingServer.java:115)
	at org.apache.thrift.server.AbstractNonblockingServer$AbstractSelectThread.handleRead(AbstractNonblockingServer.java:209)
	at org.apache.thrift.server.TNonblockingServer$SelectAcceptThread.select(TNonblockingServer.java:198)
	at org.apache.thrift.server.TNonblockingServer$SelectAcceptThread.run(TNonblockingServer.java:154)
Caused by: redis.clients.jedis.exceptions.JedisConnectionException: java.net.ConnectException: Connection refused
	at redis.clients.jedis.Connection.connect(Connection.java:134)
	at redis.clients.jedis.BinaryClient.connect(BinaryClient.java:69)
	at redis.clients.jedis.BinaryJedis.connect(BinaryJedis.java:1665)
	at com.trigonic.jedis.NamespaceJedis.connect(NamespaceJedis.java:104)
	at com.trigonic.jedis.NamespaceJedisFactory.makeObject(NamespaceJedisFactory.java:33)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1188)
	at redis.clients.util.Pool.getResource(Pool.java:20)
	... 14 more
Caused by: java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at redis.clients.jedis.Connection.connect(Connection.java:129)
	... 20 more
[ERROR] 19 Jul 11:54:50.802 AM Thread-2 [org.apache.thrift.ProcessFunction]
Internal error processing addPlanTarget

redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
	at redis.clients.util.Pool.getResource(Pool.java:22)
	at org.cz.epm.data.manager.RedisManager.getJedis(RedisManager.java:31)
	at org.cz.epm.data.manager.RedisManager.HGet(RedisManager.java:36)
	at org.cz.epm.resource.Mapper.getMapKeyValue(Mapper.java:45)
	at org.cz.epm.resource.Mapper.GetMapKey(Mapper.java:19)
	at org.cz.epm.thrift.service.EpmDatahouseImpl.addPlanTarget(EpmDatahouseImpl.java:90)
	at org.cz.epm.thrift.generated.Datahouse$Processor$addPlanTarget.getResult(Datahouse.java:1388)
	at org.cz.epm.thrift.generated.Datahouse$Processor$addPlanTarget.getResult(Datahouse.java:1)
	at org.apache.thrift.ProcessFunction.process(ProcessFunction.java:39)
	at org.apache.thrift.TBaseProcessor.process(TBaseProcessor.java:39)
	at org.apache.thrift.server.AbstractNonblockingServer$FrameBuffer.invoke(AbstractNonblockingServer.java:478)
	at org.apache.thrift.server.TNonblockingServer.requestInvoke(TNonblockingServer.java:115)
	at org.apache.thrift.server.AbstractNonblockingServer$AbstractSelectThread.handleRead(AbstractNonblockingServer.java:209)
	at org.apache.thrift.server.TNonblockingServer$SelectAcceptThread.select(TNonblockingServer.java:198)
	at org.apache.thrift.server.TNonblockingServer$SelectAcceptThread.run(TNonblockingServer.java:154)
Caused by: redis.clients.jedis.exceptions.JedisConnectionException: java.net.ConnectException: Connection refused
	at redis.clients.jedis.Connection.connect(Connection.java:134)
	at redis.clients.jedis.BinaryClient.connect(BinaryClient.java:69)
	at redis.clients.jedis.BinaryJedis.connect(BinaryJedis.java:1665)
	at com.trigonic.jedis.NamespaceJedis.connect(NamespaceJedis.java:104)
	at com.trigonic.jedis.NamespaceJedisFactory.makeObject(NamespaceJedisFactory.java:33)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1188)
	at redis.clients.util.Pool.getResource(Pool.java:20)
	... 14 more
Caused by: java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at redis.clients.jedis.Connection.connect(Connection.java:129)
	... 20 more
[ERROR] 19 Jul 11:55:17.891 AM Thread-2 [org.apache.thrift.ProcessFunction]
Internal error processing addPlanTarget

redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
	at redis.clients.util.Pool.getResource(Pool.java:22)
	at org.cz.epm.data.manager.RedisManager.getJedis(RedisManager.java:31)
	at org.cz.epm.data.manager.RedisManager.HGet(RedisManager.java:36)
	at org.cz.epm.resource.Mapper.getMapKeyValue(Mapper.java:45)
	at org.cz.epm.resource.Mapper.GetMapKey(Mapper.java:19)
	at org.cz.epm.thrift.service.EpmDatahouseImpl.addPlanTarget(EpmDatahouseImpl.java:90)
	at org.cz.epm.thrift.generated.Datahouse$Processor$addPlanTarget.getResult(Datahouse.java:1388)
	at org.cz.epm.thrift.generated.Datahouse$Processor$addPlanTarget.getResult(Datahouse.java:1)
	at org.apache.thrift.ProcessFunction.process(ProcessFunction.java:39)
	at org.apache.thrift.TBaseProcessor.process(TBaseProcessor.java:39)
	at org.apache.thrift.server.AbstractNonblockingServer$FrameBuffer.invoke(AbstractNonblockingServer.java:478)
	at org.apache.thrift.server.TNonblockingServer.requestInvoke(TNonblockingServer.java:115)
	at org.apache.thrift.server.AbstractNonblockingServer$AbstractSelectThread.handleRead(AbstractNonblockingServer.java:209)
	at org.apache.thrift.server.TNonblockingServer$SelectAcceptThread.select(TNonblockingServer.java:198)
	at org.apache.thrift.server.TNonblockingServer$SelectAcceptThread.run(TNonblockingServer.java:154)
Caused by: redis.clients.jedis.exceptions.JedisConnectionException: java.net.ConnectException: Connection refused
	at redis.clients.jedis.Connection.connect(Connection.java:134)
	at redis.clients.jedis.BinaryClient.connect(BinaryClient.java:69)
	at redis.clients.jedis.BinaryJedis.connect(BinaryJedis.java:1665)
	at com.trigonic.jedis.NamespaceJedis.connect(NamespaceJedis.java:104)
	at com.trigonic.jedis.NamespaceJedisFactory.makeObject(NamespaceJedisFactory.java:33)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1188)
	at redis.clients.util.Pool.getResource(Pool.java:20)
	... 14 more
Caused by: java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at redis.clients.jedis.Connection.connect(Connection.java:129)
	... 20 more
[ERROR] 19 Jul 11:56:55.497 AM Thread-2 [org.apache.thrift.ProcessFunction]
Internal error processing addPlanTarget

redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
	at redis.clients.util.Pool.getResource(Pool.java:22)
	at org.cz.epm.data.manager.RedisManager.getJedis(RedisManager.java:31)
	at org.cz.epm.data.manager.RedisManager.HGet(RedisManager.java:36)
	at org.cz.epm.resource.Mapper.getMapKeyValue(Mapper.java:45)
	at org.cz.epm.resource.Mapper.GetMapKey(Mapper.java:19)
	at org.cz.epm.thrift.service.EpmDatahouseImpl.addPlanTarget(EpmDatahouseImpl.java:90)
	at org.cz.epm.thrift.generated.Datahouse$Processor$addPlanTarget.getResult(Datahouse.java:1388)
	at org.cz.epm.thrift.generated.Datahouse$Processor$addPlanTarget.getResult(Datahouse.java:1)
	at org.apache.thrift.ProcessFunction.process(ProcessFunction.java:39)
	at org.apache.thrift.TBaseProcessor.process(TBaseProcessor.java:39)
	at org.apache.thrift.server.AbstractNonblockingServer$FrameBuffer.invoke(AbstractNonblockingServer.java:478)
	at org.apache.thrift.server.TNonblockingServer.requestInvoke(TNonblockingServer.java:115)
	at org.apache.thrift.server.AbstractNonblockingServer$AbstractSelectThread.handleRead(AbstractNonblockingServer.java:209)
	at org.apache.thrift.server.TNonblockingServer$SelectAcceptThread.select(TNonblockingServer.java:198)
	at org.apache.thrift.server.TNonblockingServer$SelectAcceptThread.run(TNonblockingServer.java:154)
Caused by: redis.clients.jedis.exceptions.JedisConnectionException: java.net.ConnectException: Connection refused
	at redis.clients.jedis.Connection.connect(Connection.java:134)
	at redis.clients.jedis.BinaryClient.connect(BinaryClient.java:69)
	at redis.clients.jedis.BinaryJedis.connect(BinaryJedis.java:1665)
	at com.trigonic.jedis.NamespaceJedis.connect(NamespaceJedis.java:104)
	at com.trigonic.jedis.NamespaceJedisFactory.makeObject(NamespaceJedisFactory.java:33)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1188)
	at redis.clients.util.Pool.getResource(Pool.java:20)
	... 14 more
Caused by: java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at redis.clients.jedis.Connection.connect(Connection.java:129)
	... 20 more
